### ISSUE TITLE
Some questions

### ISSUE THREAD

**Issue by gelsas** [solrachix/ebooks-manager#1]
Hi I have just stumbled upon your repository and your creation  sound awesome.

Could you give more information about which file formats are supported, do you have some screenshots of how the app looks like? 

How do I add books to this ebook manager ? 
Is it possible to automatically pull metadata from somewhere to always have complete info about the books (like Kodi,Plex, jellyfin are doing it for other media)

**Comment by gelsas** [comment]
Do you plan to add audiobook support to this app, sometimes I like to read along while listening to the audiobook.

**Comment by Leandro2585** [comment]
> Hi I have just stumbled upon your repository and your creation  sound awesome.
> 
> Could you give more information about which file formats are supported, do you have some screenshots of how the app looks like? 
> 
> How do I add books to this ebook manager ? 
> Is it possible to automatically pull metadata from somewhere to always have complete info about the books (like Kodi,Plex, jellyfin are doing it for other media) 

Hi, I have this promotional video for the app that shows you a little bit about its interface

**Comment by gelsas** [comment]
Looks great! Love this!

Could you share more information about the other questions I had as well?

**Comment by solrachix** [comment]
Hi, sorry for the delay, this is a project I started a while ago, and just now I'm going to go back to it, I'm going to create a README and put it with the images you asked for.

On some questions like this about _automatically pulling metadata_ I was thinking first about users uploading their books on the platform, and if they want, they can make them public, so other people can access them through the search field. But it's something I'm still thinking about.

**Comment by solrachix** [comment]
> Do you plan to add audiobook support to this app? Sometimes I like to read along while listening to the audiobook and having the possibility to do it from the same app would be the perfect solution.

Yes I thought I'd put this automatically, right at the upload time, but I encountered some technical issues at the time, I'll reanalyze the case again to see how it could be done.

**Comment by solrachix** [comment]
And on an image, the desktop app looks more or less like this
![image](https://user-images.githubusercontent.com/57706806/135717729-194ed869-f8b8-4b4c-a241-7a0d2214d1f0.png)
